test/unit/vdom/elements.js

```diff
@@ -1,7 +1,8 @@
 import element from '../../lib/element';
+import support from '../../../src/native/support';
 import { symbols, vdom } from '../../../src/index';
 
-const shadowDomV0 = !!document.createElement('div').createShadowRoot;
+const shadowDomV0 = !support.shadowDomV1 && support.shadowDomV0;
```

Comment:
same code here and there `src/api/vdom.js`
`!support.shadowDomV1 && support.shadowDomV0;`
can't you just import it from some common place?
LGTM otherwise